arm/Microsoft.RecoveryServices/vaults/deploy.bicep

```diff
@@ -195,7 +195,9 @@ module rsv_backupStorageConfiguration 'backupStorageConfig/deploy.bicep' = if (!
     recoveryVaultName: rsv.name
     storageModelType: backupStorageConfig.storageModelType
     crossRegionRestoreFlag: backupStorageConfig.crossRegionRestoreFlag
-    enableDefaultTelemetry: enableReferencedModulesTelemetry
+    enableDefaultTelemetry: enableChildTelemetry
```

Comment:
Please replace any occurrence of `enableChildTelemetry` with `enableReferencedModulesTelemetry`

Reply:
Done